lib/ansible/modules/network/netact/netact_cm_command.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/python
+# Copyright: Nokia
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# pylint: disable=invalid-name
+# pylint: disable=wrong-import-position
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-branches
+# pylint: disable=too-many-statements
+
+"""
+NetAct CM ansible command module
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: netact_cm_command
+
+short_description: NetAct CM operation ansible module
+
+version_added: "2.5"
+
+description:
+    netact_cm_command can be used to run various configuration management operations.
+    This ansible module requires that target host have Nokia NetAct Configuration Management.
+    This module requires that the target hosts have Nokia NetAct network management system installed.
+    Module will access the Configurator command line interface in NetAct to upload network configuration to NetAct,
+    run configuration export, plan import and configuration provision operations
+    To set the scope of the operation, define Distinguished Name (DN) or Working Set (WS) or
+    Maintenance Region (MR) as input.
+options:
+    operation:
+        description:
+            operation
```

Comment:
Would be good change this line to explain a bit about what this is done.

No need to detail the supported operations, that's what `choices:` is for, so the following line can be removed.

Reply:
I will add more detail description to here